storage-resize-images/POSTINSTALL.md

```diff
@@ -12,13 +12,24 @@ You can test out this extension right away:
 
 ### Using the extension
 
+You can upload images using the [Cloud Storage for Firebase SDK](https://firebase.google.com/docs/storage/) for your platform (iOS, Android, or Web). Alternatively, you can upload images directly in the Firebase console's Storage dashboard.
+
 When you upload an image file to `${param:IMG_BUCKET}`, this extension:
 
 - Creates resized image(s) with your specfied dimensions.
-- Stores the resized image(s) in the bucket `${param:IMG_BUCKET}` (and, if configured, under the path `${param:RESIZED_IMAGES_PATH}`).
 - Names resized image(s) using the same name as the original uploaded image, but suffixed with the specified width and height.
+- Stores the resized image(s) in the bucket `${param:IMG_BUCKET}` (and, if configured, under the path `${param:RESIZED_IMAGES_PATH}`).
 
-You can upload images using the [Cloud Storage for Firebase SDK](https://firebase.google.com/docs/storage/) for your platform (iOS, Android, or Web). Alternatively, you can upload images directly in the Firebase console's Storage dashboard.
+The extension also copies over to the resized image(s) the following [metadata](https://cloud.google.com/storage/docs/json_api/v1/objects/insert#request_properties_JSON) associated with the original image:
+
+- content disposition
+- content encoding
+- content language
+- content type
+- user-provided metadata (except Firebase storage download tokens)
+- Cache-Control header
```

Comment:
I think these should either match the [docs](https://cloud.google.com/storage/docs/json_api/v1/objects/insert#request_properties_JSON) exactly ("content disposition" -> "contentDisposition"), or match the IETF spec names ("content disposition" -> "[Content-Disposition](https://tools.ietf.org/html/rfc6266)") so that it is as easy as possible for a developer to figure out what each header does.

Reply:
Cool - agree.  I've changed them to the IETF spec names and added links to the mozilla dev docs for each.  See the recent commit.